### DISCUSSION TITLE
Deal with long cookie session

### DISCUSSION THREAD

**Discussion by netdesignr** [sergiodxa/remix-auth#226] [Q&A]
Hi,

Can this package handle large session cookie if it becomes too big? The browser by default will reject larger cookie than a specific size. Unfortunately, because the JWT has to have a lot of info in it there's no way the information can be stripped out. Is there any way this package handle this type of situations? I could see that nextauth have introduced that in their more recent version.

I want to move from NextJS to Remix but the Auth with Okta and handling large cookie session is the most important thing that the new stack has to handle.

Any thoughts?

Cheers!
Mihai

**Comment by sergiodxa** [comment]
Remix Auth uses Remix session and cookie features, Remix Auth it doesn't do anything related to that.

But if your session stores a value larger than the cookie size limit you can change from createCookieSessionStorage to another session storage (e.g. Worker KV or Redis) to store the data outside the cookie.
